dysnix/proxysql/Chart.yaml

```diff
@@ -2,10 +2,10 @@ apiVersion: v1
 appVersion: "2.5.5"
 description: ProxySQL Helm chart for Kubernetes
 name: proxysql
-version: 0.10.3-splashthat-rc1
+version: 0.10.3-splashthat-rc2
 home: https://www.proxysql.com/
 sources:
-  - https://github.com/dysnix/charts
+  - https://github.com/SplashThat/dysnix-charts
```

Comment:
can you explain this change?

Reply:
I asked github to broke the link between our repository and the source one because our proxysql charts derived too much from it.
So instead of saying that the source of this proxysql helm chart comes from dysnix charts repository, I changed it for ours